webserver/Models/Companies/Company.cs

```diff
@@ -1,15 +1,10 @@
-﻿using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
-using Microsoft.AspNetCore.Identity;
+﻿using Microsoft.AspNetCore.Identity;
+using System.ComponentModel.DataAnnotations;
 
 namespace webserver.Models;
 
 public class Company : IdentityUser {
```

Comment:
The company is not a user.
A user represents a single entity that can log in in a system.

A company is a generic entity, which can be represented by a user.